lib/MultiColumnList/MCLRenderer.js

```diff
@@ -674,6 +674,12 @@ class MCLRenderer extends React.Component {
     }
   };
 
+  handleRowKeyPress = (e, row) => {
+    if (e.keyCode === 13 || e.keyCode === 32) {
+      this.handleRowClick(e, row);
+    }
+  }
```

Comment:
We need a conditional here to see if the target was within the row... if so, the onRowClick event shouldn't be triggered.

```suggestion
  handleRowKeyPress = (e, row) => {
    if (!e.target.matches(`.${css.mclRow} *`)) {
      if (e.keyCode === 13 || e.keyCode === 32) {
        this.handleRowClick(e, row);
      }
    }
  }
```

Again, thank you for your effort on this! I'm sorry the process has taken longer... but it may take longer still to shore up any tests that counted on a `<button>` being there...